test/spec/services/statServicesSpec.js

```diff
@@ -138,6 +138,23 @@ describe('Stats Service', function() {
     });
   });
 
+  describe('canLevelUp', function() {
+  	it('returns true if a user has enough exp to level up', function() {
+  		user.stats.exp = 150;
```

Comment:
The other tests use magic numbers -- as a side note these should really be lookups based on a dictionary of how much exp is needed to get to level 2. If it ever changes these tests will all fail and to anyone making new or more nuanced tests it is a lot harder to figure out what magic numbers to use.